README.md

```diff
@@ -1050,8 +1050,10 @@ The `DatabaseLogging` middleware checks for a logger injected in the request
 which in turn uses the logger to produce database query logs. A nice
 side-effect of this approach is that, if the logger is tagged with a
 `request_id`, there's a logs correlation between the HTTP requests and the
-database queries. Also, if the logger is tagged with `treace_id` we can easily
-correlate logs with traces and see corresponding database queries.
+database queries. Also, if the logger is tagged with `trace_id` we can easily
+correlate logs with traces and see corresponding database queries. Have in mind 
+that ORM logging happens with the same level as it's based logger but with include
+database fields ('elapsed_seconds', 'affected_rows' and 'sql') only on `debug` mode.
```

Comment:
Nice catch on the `treace_id` typo. 🦅 

```suggestion
database queries. Also, if the logger is tagged with `trace_id` we can easily
correlate logs with traces and see corresponding database queries. Keep in mind 
that ORM logging happens at the same level as the base logger. The additional
database fields ('elapsed_milliseconds', 'affected_rows' and 'sql') are available
only when using the `debug` or `trace` levels.
```

☝️ this might have to be revised again before we merge the PR.